src/lib/Preview.js

```diff
@@ -795,7 +795,10 @@ class Preview extends EventEmitter {
         // Set the authorization token
         this.options.token = tokenMap[this.file.id];
 
-        this.setupUI();
+        // Do nothing if container is already setup and in the middle of retrying
+        if (!(this.ui.isSetup() && this.retryCount > 0)) {
+            this.setupUI();
```

Comment:
Do we want to add this to the `reload` case? We call it a few times throughout preview.js

Reply:
I'm not sure, IMO the reload intention is to wipe everything clean and start over from scratch in which case, it seems to make sense to always re setup the UI